rust-mode.el

```diff
@@ -170,7 +170,18 @@ function or trait.  When nil, where will be aligned with fn or trait."
   "Face for the question mark operator."
   :group 'rust-mode)
 
+(defface rust-builtin-formatting-macro-face
+  '((t :inherit font-lock-builtin-face))
+  "Face for builtin formatting macros \(print! &c.\)."
```

Comment:
Ah, I didn’t know that. I got into the habit of escaping them a long time ago to avoid issues with matching. Thanks!